server/storage.lua

```diff
@@ -6,16 +6,13 @@
 
 ---@param request InsertVehicleEntityRequest
 function InsertVehicleEntity(request)
-    MySQL.insert('INSERT INTO player_vehicles (license, citizenid, vehicle, hash, mods, plate, garage, state) VALUES (?, ?, ?, ?, ?, ?, ?, ?)', {
-        request.license,
-        request.citizenId,
-        request.model,
-        joaat(request.model),
-        '{}',
-        request.plate,
-        'pillboxgarage',
-        0
+    exports.qbx_vehicles:CreateVehicleEntity({
+        citizenId = request.citizenId,
+        model = request.model,
+        plate = request.plate,
+        state = 0
     })
+    return vehicleId
```

Comment:
Where does this variable come from? Also InsertVehicleEntity doesn't seem like it needs to exist only as a wrapper of CreateVehicleEntity. Consider moving CreateVehicleEntity in-line.

Reply:
woopsies, idk about moving the export directly in main.lua, feels like storage is kinda made for this 🤔

Reply:
It's an unneeded abstraction, no? The same data passed two levels down? Doesn't feel like this function is doing any additional work.